### ISSUE TITLE
Undefined variable: OC_Version at /var/www/owncloud/lib/private/util.php#325

### ISSUE THREAD

**Issue by fluffy88** [owncloud/core#14741]
### Description
My ownCloud installation is constantly generating a number of error logs.  However I have not noticed any negative functional effects as a result of these logs.  I think the errors may be generated by the ownCloud cron.

### Steps to reproduce
I'm not exactly sure what is causing the issue, I first noticed the error logs after manually upgrading from ownCloud 7.0.4 to 8.0.  

### Expected behaviour
Nothing :)
ownCloud should run without outputting error logs

### Actual behaviour
I get a large number of the same 5 logs.  See logs section below for actual logs.

### Server configuration
**Operating system:** Rasbian

**Web server:** nginx/1.2.1

**Database:** sqlite3

**PHP version:** PHP 5.4.36-0+deb7u3

**ownCloud version:** 8.0.0.7

**Updated from an older ownCloud or fresh install:** Manual upgrade from 7.0.4

**List of activated apps:**
User installed apps
* calendar
* contacts

Default apps
* activity
* files
* files_sharing
* files_trashbin
* files_versions
* files_videoviewer
* gallery
* templateeditor
* updater

**The content of config/config.php:**

```
  'instanceid' => 'ocbdbf0c6e27',
  'passwordsalt' => 'hmmmm yummy salt',
  'secret' => 'shh its a secret',
  'trusted_domains' =>
  array (
    0 => 'fqdn',
  ),
  'datadirectory' => '/not/default/dir/owncloud/data',
  'overwrite.cli.url' => 'https://fqdn/owncloud',
  'dbtype' => 'sqlite3',
  'version' => '8.0.0.7',
  'installed' => true,
  'overwritewebroot' => '/owncloud',
  'theme' => '',
  'maintenance' => false,
  'asset-pipeline.enabled' => true,
  'forcessl' => false,
  'mail_smtpmode' => 'php',
```

**Are you using external storage, if yes which one:** My owncloud/data directory is on a USB external HDD which is mounted locally, in a directory outside the webservers web directory.

**Are you using encryption:** No

**Are you using an external user-backend, if yes which one:** No

#### ownCloud log (data/owncloud.log)
```
{"reqId":"e0db713a3b5eb519f4234d78826811d2","remoteAddr":"","app":"PHP","message":"Undefined variable: OC_Version at \/var\/www\/owncloud\/lib\/private\/util.php#325","level":3,"time":"2015-03-05T20:15:02+00:00"}
{"reqId":"e0db713a3b5eb519f4234d78826811d2","remoteAddr":"","app":"PHP","message":"Undefined variable: OC_VersionString at \/var\/www\/owncloud\/lib\/private\/util.php#327","level":3,"time":"2015-03-05T20:15:02+00:00"}
{"reqId":"e0db713a3b5eb519f4234d78826811d2","remoteAddr":"","app":"PHP","message":"Undefined variable: OC_Channel at \/var\/www\/owncloud\/lib\/private\/util.php#329","level":3,"time":"2015-03-05T20:15:02+00:00"}
{"reqId":"e0db713a3b5eb519f4234d78826811d2","remoteAddr":"","app":"PHP","message":"Undefined variable: OC_Build at \/var\/www\/owncloud\/lib\/private\/util.php#331","level":3,"time":"2015-03-05T20:15:02+00:00"}
{"reqId":"e0db713a3b5eb519f4234d78826811d2","remoteAddr":"","app":"PHP","message":"implode(): Invalid arguments passed at \/var\/www\/owncloud\/lib\/private\/util.php#1382","level":3,"time":"2015-03-05T20:15:02+00:00"}
```

**Comment by PVince81** [comment]
Can you describe your manual upgrade process ? It is likely that something went wrong and you might have a mix with old code. Did you delete everything except config and days folders, then extracted the new tarball?

**Comment by fluffy88** [comment]
Hi PVince81,

Thanks for the quick reply.  I actually had the whole thing typed out and accidentally closed the browser :/ so in my re-typed description I forgot to add the link to the guide I followed.  This is the guide I followed, https://doc.owncloud.org/server/8.0/admin_manual/maintenance/upgrade.html.
The basic procedure was, I backed up my old owncloud/ dir, extracted the tarball and copied back in the config file.  My data/ dir was in a separate location, so it remained unchanged.

I have investigated the issue a little and found that the `lib/private/util.php` function [loadVersion](https://github.com/owncloud/core/blob/master/lib/private/util.php#L324) is requiring `version.php` which is located in the ownCloud root dir.  It seems the require statement `require 'version.php';` isn't working.

I have modified the require statement to `require OC::$SERVERROOT . '/version.php';` and the errors have went away.
